mail/src/main/java/module-info.java

```diff
@@ -49,4 +45,7 @@
             com.sun.mail.smtp.SMTPSSLProvider,
             com.sun.mail.pop3.POP3Provider,
             com.sun.mail.pop3.POP3SSLProvider;
+    uses jakarta.mail.util.StreamProvider;
```

Comment:
this is needed when the module does the lookup for this service - I do not see it's the case here, so this line should be removed